pointerevents/pointerevent_support.js

```diff
@@ -185,3 +193,49 @@ function rPointerCapture(e) {
     catch(e) {
     }
 }
+
+var globalPointerEventTest = null;
+var expectedPointerType = null;
+
+var HOVERABLE_POINTERS = ['mouse', 'pen'];
+
+function MultiPointerTypeTest(testName, types) {
+    this.testName = testName;
+    this.types = types;
+    this.currentTypeIndex = 0;
+    this.createNextTest();
+}
+
+MultiPointerTypeTest.prototype.skip = function() {
+    this.createNextTest().timeout();
```

Comment:
I find returning the previous test to be counter-intuitive.  When I first read this I thought "why does calling skip cause the NEXT text to timeout?".

Is the order (eg. invoking timeout() on the last test AFTER creating the new test) actually required?  Or could you just change this to, eg.:
```javascript
  this.currentTest.timeout();
  this.createNextTest();
```

If you want to just be explicit and name this function `createNextTestAndReturnPreviousTest` or something like that, that would be enough to address my concern.  But hopefully we can just avoid having to operate on the previous test after creating the new test.